### PR TITLE
Use padding

### PR DESCRIPTION
**Pull request by treyhoover**
This could be an optional configuration as well if you'd like.

### CODE REVIEW

**Comment by elliottsj** [comment]
Thanks for the PR!

I'd prefer if this was configurable. Can you update the signature of `new GenerateJsonPlugin` to match `JSON.stringify`? e.g.
```js
new GenerateJsonPlugin(
  'my-file.json',
  /* value: */ { foo: 'bar' },
  /* replacer: */ null,
  /* space: */ 2
)
```

**Comment by treyhoover** [comment]
Sure thing! Should the default be 2 or 0?

**Comment by elliottsj** [comment]
Default should be the same as [`JSON.stringify`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify): if no 'space' argument is provided (i.e. `undefined`), then no whitespace is used

**Comment by treyhoover** [comment]
How's that?

**Comment by elliottsj** [comment]
e.g. if you use
```js
new GenerateJsonPlugin(
  'my-file.json',
  /* value: */ { foo: 'bar' }
)
```

then there's no 3rd & 4th argument (the 4th argument being the number of spaces). So the plugin should call `JSON.stringify({ foo: 'bar' })` (equivalently `JSON.stringify({ foo: 'bar' }, undefined, undefined)`)

File: index.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
-function GenerateJsonPlugin(filename, obj) {
-  this.filename = filename;
-  this.obj = obj;
+function GenerateJsonPlugin(filename, { value, replacer, space }) {
```
Comment (elliottsj) [inline index.js]: Can you make this
```js
function GenerateJsonPlugin(filename, value, replacer, space) {
```

after that, looks good!

**Comment by treyhoover** [comment]
Sorry, I meant how's the PR now that I pushed those changes, haha. I think we're on the same page.

File: index.js (hunk 1 above)
Comment (treyhoover) [inline index.js]: Sure, can do.

**Comment by treyhoover** [comment]
Updated

**Comment by elliottsj** [comment]
Awesome, thanks 😄

**Comment by elliottsj** [comment]
Published as v0.2.0

**Pull request merged**